app/assets/stylesheets/components/_repo-list.scss

```diff
@@ -0,0 +1,67 @@
+.repo-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -$base-spacing;
+  padding-top: 1.25rem;
+}
+
+.repo-item {
+  background-color: darken($base-background-color, 5%);
+  border-radius: $base-border-radius;
+  box-shadow: 0 1px 0 lighten($base-background-color, 5%);
+  flex-basis: 300px;
+  flex-grow: 1;
+  margin-bottom: $base-spacing;
+  margin-right: $base-spacing;
+  overflow: hidden;
+  padding: $base-spacing ($small-spacing * 1.5) 3rem;
+  position: relative;
+}
+
+.repo-item-header {
+  @include clearfix;
+  background-color: $codetriage-blue;
+  margin: (-$base-spacing) (-$small-spacing * 1.5) $small-spacing;
+  padding: $small-spacing ($small-spacing * 1.5);
+
+  .high & {
+    background-color: $codetriage-red;
+  }
+
+  .medium & {
+    background-color: $codetriage-yellow;
+  }
+
+  .low & {
+    background-color: $codetriage-green;
+  }
+}
+
+.repo-item-title {
+  color: $base-font-color;
+  float: left;
+  font-size: 1.125rem;
+  margin-bottom: 0;
```

Comment:
Should all headings have no bottom margin by default? Looks like you've done this a couple times